### ISSUE TITLE
"Use template name directly" points to wrong template file

### ISSUE THREAD

**Issue by untitaker** [sunng87/handlebars-rust#98]
Steps to reproduce:
1. Modify `examples/base0.hbs` such that the block name is incorrectly quoted. See https://gist.github.com/untitaker/87311dc36dd42f1791542b7dc61a968f
2. Run `cargo run --example partials`

The example panics as expected. However, when modifying `partials.rs` such that the error message is printed, the following is printed:

```
Err(RenderError { desc: "Do not use literal here, use template name directly.", template_name: Some("template"), line_no: Some(5), column_no: Some(5) })
```

There are two thing wrong with this:
- "page" is not a template name
- The line/column information should point to `base0`.

**Comment by sunng87** [comment]
Thank you @untitaker . Currently the error report mechanism cannot trace into included templates. I will leave it for future fix.

**Comment by sunng87** [comment]
This is fixed in #121 , both string and ref is supported in `include`
